workflow-dev/tpl/deis-logger-rc.yaml

```diff
@@ -19,6 +19,22 @@ spec:
       - name: deis-logger
         image: quay.io/{{.logger.org}}/logger:{{env "LOGGER_GIT_TAG" | default .logger.dockerTag}}
         imagePullPolicy: {{.logger.pullPolicy}}
+        env:
+        - name: STORAGE_ADAPTER
+          value: redis
+{{- if or (eq (env "LOGGER_REDIS_LOCATION") "off-cluster") (eq .logger_redis_location "off-cluster") }}
+        - name: DEIS_LOGGER_REDIS_DB
+          value: "{{ or (env "LOGGER_REDIS_DB") .loggerRedis.db }}"
+        - name: DEIS_LOGGER_REDIS_SERVICE_HOST
+          value: "{{ or (env "LOGGER_REDIS_HOST") .loggerRedis.host }}"
+        - name: DEIS_LOGGER_REDIS_SERVICE_PORT
+          value: "{{ or (env "LOGGER_REDIS_PORT") .loggerRedis.port }}"
```

Comment:
is this different from `{{env "LOGGER_REDIS_PORT"  | default .loggerRedis.port}}`?